### ISSUE TITLE
Java version increase

### ISSUE THREAD

**Issue by mbfakourii** [Tkko/flutter_smart_auth#35]
https://github.com/Tkko/flutter_smart_auth/blob/2786f7b8cba8b9033fd8223bb8d88e2ff300614d/android/build.gradle#L36

Why is the java version so high?
Many packages are not yet on this version and we are having problems in most projects.

I suggest setting it to the following values

```dart
compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
}

kotlinOptions {
    jvmTarget = '1.8'
}
```

**Comment by Tkko** [comment]
Hi @mbfakourii, good point. The official packages like `SharedPreferences` and `ImagePicker` are using VERSION_11 so it's better to set the same version, WDYT?

**Comment by Tkko** [comment]
@mbfakourii  here is the [PR](https://github.com/Tkko/flutter_smart_auth/pull/36) you can review if you want.

**Comment by Tkko** [comment]
Fixed in v3.1.0
